packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/CameraPermissions.java

```diff
@@ -90,9 +93,9 @@ public boolean onRequestPermissionsResult(int id, String[] permissions, int[] gr
 
       alreadyCalled = true;
       if (grantResults[0] != PackageManager.PERMISSION_GRANTED) {
-        callback.onResult("cameraPermission", "MediaRecorderCamera permission not granted");
+        callback.onResult("CameraAccessDenied", "Camera access permission was denied.");
```

Comment:
It would be helpful to declare a set of string constants in a single location with a reference to the Dart code that handles these, so that it's obvious to maintainers (by looking at existing examples) that they shouldn't just be using random string values when adding new errors.